lib/index.js

```diff
@@ -27,6 +27,17 @@ class PWMetrics {
     this.sheets = opts.sheets;
     this.expectations = opts.expectations || false;
     this.tryLighthouseCounter = 0;
+    this.desiredMetrics = [];
```

Comment:
This is here to ensure that the .length property of [this.desiredMetrics](https://github.com/pedro93/pwmetrics/blob/145b489f98636a5d37c7f2e5fe8523deed965f38/lib/index.js#L40) does not crash. Imagine you are running pwmetrics as a node module and don't pass the desiredMetrics, the code must have a fallback.